Orange/widgets/tests/base.py

```diff
@@ -364,6 +364,26 @@ def modifiers(self, modifiers):
         finally:
             QTest.keyRelease(self.widget, Qt.Key_BassBoost, old_modifiers)
 
+    def test_minimum_size(self):
+        widget = getattr(self, "widget", None)
+        if widget is None:
+            return
+
+        def invalidate_cached_size_hint(w):
+            # as in OWWidget.setVisible
+            if w.controlArea is not None:
+                w.controlArea.updateGeometry()
+            if w.buttonsArea is not None:
+                w.buttonsArea.updateGeometry()
+            if w.mainArea is not None:
+                w.mainArea.updateGeometry()
+
+        invalidate_cached_size_hint(widget)
+        min_size = widget.minimumSizeHint()
+        # FIXME high-dpi screens
```

Comment:
On mac, size returns size in virtual, not device pixels, so the values you have are ok.